src/App.js

```diff
@@ -46,27 +49,49 @@ const theme = createTheme({
 
 export default function App() {
   const [devMode, setDevMode] = React.useState(getIsDevMode);
+  const [userState, setUserState] = React.useState({
+    userName: "",
+    isLoggedIn: false,
+  });
+
+  const refresh = () => {
```

Comment:
better to have this in the useEffect, that way is simpler to keep track of the dependencies

Reply:
Thanks for the feedback. The goal is to allow checking the login state at some key point:
- When the user arrives on the page
- When the user answered its 3 first questions (to ask him to log in if not yet)

After we can think about a cleaver way to check cookies, like when not logged in every 3s check is the cookies changed. But that's not the main problem for now.

I'm merging because we do not have the possibility to check that it's working except in production due to some CORS issues